gateway-server/src/test/java/org/apache/knox/gateway/services/token/impl/DefaultTokenStateServiceTest.java

```diff
@@ -369,16 +370,7 @@ protected TokenStateService createTokenStateService() throws Exception {
 
   /* create a test JWT token */
   protected JWT getJWTToken(final long expiry) {
-    String[] claims = new String[6];
-    claims[0] = "KNOXSSO";
-    claims[1] = "john.doe@example.com";
-    claims[2] = "https://login.example.com";
-    if(expiry > 0) {
-      claims[3] = Long.toString(expiry);
-    }
-    claims[4] = "E0LDZulQ0XE_otJ5aoQtQu-RnXv8hU-M9U4dD7vDioA";
-    claims[5] = null;
-    JWT token = new JWTToken("RS256", claims);
+    JWT token = new JWTToken(new JWTokenAttributesBuilder().setExpires(expiry).setAlgorithm("RS256").build());
```

Comment:
This test is slightly different in that the test token has differing contents. Has the coverage of this test changed as a result? Or, does it only reflect the change in handling of claims, whereas now, previously necessary claims are no longer necessary?

Reply:
Putting it back or removing it makes no difference. These attributes are not checked in this test. `JWTTokenTest` covers the attribute checks.